docs/SUMMARY.md

```diff
@@ -64,7 +65,7 @@
         - [Payload format](dev/core/internal/payload.md).
         - [Directory structure](dev/core/internal/directory-structure.md)
         - [Debug Pings](dev/core/internal/debug-pings.md)
-    - [Howtos](dev/howtos/index.md)
+    - [HOWTOs](dev/howtos/index.md)
```

Comment:
Not a big fan of all-uppercasing this (but also not a blocker)

Reply:
Yeah -- I'll revert this one (and add it to the dictionary).